### ISSUE TITLE
High RAM usage until crash

### ISSUE THREAD

**Issue by Vitaaliyy** [henkelmax/camera#76]
**Please read the [FAQ](https://modrepo.de/minecraft/camera/faq) before submitting a bug report!**

**Bug description**
It seems like one of the files of the mod uses too much RAM, I have yet to change the JMV garbage collector arguments, maybe it will fix it but I think the mod is responsible for the recent error 137 crashes we've been experiencing on our server. 

Do you know of a way to reduce RAM usage with your mod? It stays constant for a bit and goes up in huge bits, almost 1gb per 1gb sometimes, until it crashes with error 137.

 When looking at the heap dump analysis, it says the cause is a file called "class de.maxhenkel.camera.Main @ 0x6283cbd80".

**Steps to reproduce the issue**
1. Play on server with mod
2. Ram usage spikes up sometimes, and never goes down, until it crashes. 
3. Checking heap dumb it says it's bc of a file related to the mod.

**Expected behavior**
Not use so much ram.

**Log files**
No crash reports with error 137 but here's the heap dump analysis https://heaphero.io/heap-report-wc.jsp?p=Yk0zQytqQTQvSU5SeWFxcXRVUTYxTzdPa0hzdFJObGpldzRXVk9IdmVkclRLSWY4bjI3czhaT0dNSDFsOGd5ZnpoYWFKVGp5OHU2U0pWMFVZVG9FMFhtYjlUUHF5MmJPb2Q5QXZQOXNZQTQ9#ls_section 

**Versions**
- Minecraft version: 1.18.2
- Forge version: 40.2.0
- Mod version: 1.0.5

**Other mods**
Link to my modpack, you can just open the modlist: https://drive.google.com/file/d/1hgScJNDm4UM0xxkXRDeUJnsn6u7b5pXt/view?usp=sharing

**Comment by henkelmax** [comment]
Do you have an exceptional amount of images on the server?

**Comment by Vitaaliyy** [comment]
Umm, so I checked the world files... But, I'm not sure if the pictures in the world files are exclusively ones that are still on the server, are deleted ones kept in the "camera_images" folder? 

That being said, we have 1455 files in that folder... ummm.. Is that an "exceptional amount" to you or? 

Maybe we got too excited with your mod, it's awesome sorry. 

What can we do to reduce how many images, will deleting the ones in the "camera_images" folder just remove them from the server? 

Thank you for your answer!
